config/admin-functions/create-machine-cert.sh

```diff
@@ -35,65 +32,21 @@ function get_machine_jurisdiction_from_user_input() {
     echo "${machine_jurisdiction}"
 }
 
-# USB helpers
-
-# Returns the path to the first connected USB stick, if any. Returns an empty string if no USB
-# stick is connected.
-function get_usb_path() {
-    lsblk /dev/disk/by-id/usb*part* --noheadings --output PATH 2> /dev/null | grep / --max-count 1
-}
-
-# Errs if no USB is mounted (or if unmounting fails for some other reason)
-function unmount_usb() {
-    "${VX_METADATA_ROOT}/app-scripts/unmount-usb.sh"
-}
-
-# Never errs, even if no USB is mounted
-function unmount_usb_if_mounted() {
-    "${VX_METADATA_ROOT}/app-scripts/unmount-usb.sh" 2> /dev/null || true
-}
-
-function mount_usb() {
-    local usb_path="${1}"
-    unmount_usb_if_mounted
-    "${VX_METADATA_ROOT}/app-scripts/mount-usb.sh" "${usb_path}"
-}
-
-function mount_usb_if_present() {
-    local usb_path="$(get_usb_path)"
-    if [[ -n "${usb_path}" ]]; then
-        mount_usb "${usb_path}"
-    fi
-}
-
-function wait_for_usb_and_mount_once_present() {
-    while true; do
-        local usb_path="$(get_usb_path)"
-        if [[ -n "${usb_path}" ]]; then
-            mount_usb "${usb_path}"
-            break
-        fi
-        sleep 1
-    done
-}
-
-# OpenSSL helpers
-
 function create_machine_cert_signing_request() {
-    pushd "${VX_METADATA_ROOT}/vxsuite/libs/auth" > /dev/null
     local machine_jurisdiction="${1:-}"
     if [[ -n "${machine_jurisdiction}" ]]; then
         VX_MACHINE_TYPE="${VX_MACHINE_TYPE}" \
         VX_MACHINE_JURISDICTION="${machine_jurisdiction}" \
-        ./scripts/create-production-machine-cert-signing-request
+        "${VX_METADATA_ROOT}/vxsuite/libs/auth/scripts/create-production-machine-cert-signing-request"
     else
         VX_MACHINE_TYPE="${VX_MACHINE_TYPE}" \
-        ./scripts/create-production-machine-cert-signing-request
+        "${VX_METADATA_ROOT}/vxsuite/libs/auth/scripts/create-production-machine-cert-signing-request"
```

Comment:
These scripts used to have to be run from `vxsuite/libs/auth` but can now be run from anywhere (https://github.com/votingworks/vxsuite/pull/4178)